src/repository/base.repository.js

```diff
@@ -34,10 +34,43 @@ class BaseRepository {
    * @returns {object} - returns an database object
    * @memberof BaseRepository
    */
-  static async findOneByField(model, options) {
+  static findOneByField(model, options) {
     return model.findOne({ where: options });
   }
 
+  /**
+   * @static
+   * @param {*} model
+   * @param {*} options
+   * @returns {object} - returns an database object
+   * @memberof BaseRepository
+   */
+  static findOne(model, options) {
```

Comment:
These methods `findAll`, `updateField` already exist on the database. Also, I noticed you changed a lot of the methods already in the base repository. Could you please return them all to the way they were as many implementations depend on them

Reply:
Thank you for this review.
Recall that you suggested earlier that I adhere to the methods already created in the `baserespository`, are you now saying that I should go back to use the sequelize methods directly on the controller?